.github/workflows/e2e-tests.yml

```diff
@@ -133,20 +133,17 @@ jobs:
         id: deps-install
         run: npm install
 
-      - name: Cache Playwright Browsers
+      - name: Cache browsers
         uses: actions/cache@v4
         with:
           path: |
             ~/.cache/ms-playwright/
           key: ${{ runner.os }}-playwright-${{ hashFiles('package-lock.json') }}
 
-      - name: Install Playwright Browsers
+      - name: Install browsers
         run: npm run playwright install --with-deps
 
-      - name: build web
-        run: npm run tronb:vite:dev
```

Comment:
Why was this ever required? Snapshots don't rely on the Electron build, do they?